### PR TITLE
Unidecode: Convert to use template

### PR DESCRIPTION
**Pull request by marianosimone**
Just converting the IA to return a structured_answer and use templates

Instant Answer Page: 

## Unidecode: Convert to use template

This is just converting the IA to use a template instead of returning HTML

## Related Issues and Discussions

Part of #1163

## People to notify

@pjhampton @moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/unidecode

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
Hey @marianosimone

Can you please use our [PR Template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/.github/PULL_REQUEST_TEMPLATE.md)? Ping me once you have updated and I will reopen this.

**Comment by marianosimone** [comment]
@pjhampton I think that should do it

File: t/Unidecode.t [hunk 1]
```diff
@@ -9,14 +9,25 @@ use DDG::Test::Goodie;
...
+}
 
 ddg_goodie_test(
 	[qw(
 		DDG::Goodie::Unidecode
 	)],
-	'unidecode møæp'   => test_zci('moaep'),
-    "unidecode \x{5317}\x{4EB0}" => test_zci('Bei Jing'),
-    'unidecode åäº°' => test_zci('aaodeg'),
+	'unidecode møæp'  => build_test('moaep'),
```
Comment (mintsoft) [inline t/Unidecode.t]: There's an "evil" hard-tab here!

Comment (marianosimone) [inline t/Unidecode.t]: Fixed! I also made the " and ' consistent

**Comment by gautamkrishnar** [comment]
@marianosimone tests are failing... please fix it...

**Comment by marianosimone** [comment]
@gautamkrishnar: done!

Forgot that ' and " are not the same in Pearl :(

**Comment by mintsoft** [comment]
@marianosimone Try `qq''` vs `q""` :laughing:

**Comment by gautamkrishnar** [comment]
Thanks a lot for contributing :heart:

**Pull request merged**